HGLRCF411.config

```diff
@@ -0,0 +1,105 @@
+ # Betaflight / STM32F411 (S411) 4.0.0 Apr 11 2019 / 07:44:46 (9ad2cc844) MSP API: 1.41
```

Comment:
This needs to go into `unified_targets/configs`.

Reply:
I will modify it right away.